commons-vfs2/src/test/java/org/apache/commons/vfs2/AbstractProviderTestCase.java

```diff
@@ -114,14 +114,22 @@ protected void assertSameContent(final String expected, final FileObject file) t
      * are encoded using UTF-8.
      */
     protected void assertSameURLContent(final String expected, final URLConnection connection) throws Exception {
+        assertSameURLContent(expected, connection.getInputStream(), connection);
```

Comment:
This does not look right, the method that allocates the resource should release it.

Reply:
I can add an additional try-catch however....